util/solver/executor/benchmarks.py

```
@@ -0,0 +1,186 @@
+                token in line.lower() for token in filter_tokens
+            ):
+                continue
+            fields = line.split()
+            if len(fields) < 2:
+                continue
+            yield fields
+
+
+def locate_checkpoint(checkpoint_root: str, checkpoint_fragment: str) -> str:
+    for suffix in ("gz", "zstd"):
+        result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)